### PR TITLE
Basic javascript

### PR DESCRIPTION
**Pull request by razarehan28**

### CODE REVIEW

File: BOM JavaScript/bom.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
+// Window Height and Width
+// innerHeight
+function resizeFunction() { // dont need to refresh
+	console.clear();
+	var inHeight = window.innerHeight; // innerHeight : 880
+	console.log('innerHeight : ' + inHeight);
+	// outerHeight
+	var ouHeight = window.outerHeight; // outerHeight : 985
```
Comment (desaianiruddh) [inline BOM JavaScript/bom.js]: Use proper variable name. Here you should use outerHeight

File: BOM JavaScript/bom.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
+// Window Height and Width
```
Comment (desaianiruddh) [inline BOM JavaScript/bom.js:1]: set indentation tab size 2 instead of 4

File: BOM JavaScript/bom.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+// innerHeight
+function resizeFunction() { // dont need to refresh
+	console.clear();
+	var inHeight = window.innerHeight; // innerHeight : 880
```
Comment (desaianiruddh) [inline BOM JavaScript/bom.js]: use let instead of var

File: HtmlDOM/dom.js [hunk 4]
```diff
@@ -21,7 +21,7 @@ console.log(element);
 function hello() {
 	alert("Hello everyone");
 }
-EventListener
+//EventListener
 document.getElementById("heading").addEventListener
 	("click", function (e) {
```
Comment (desaianiruddh) [inline HtmlDOM/dom.js:26]: needs to improve code indentation

**Review by sheth-jay — APPROVED** [review]

**Pull request merged**